ngboost/distns/betabernoulli.py

```diff
@@ -0,0 +1,125 @@
+from scipy.stats import betabinom as dist
+from scipy.stats import beta as betadist
+import numpy as np
+from ngboost.distns.distn import RegressionDistn
+from ngboost.scores import LogScore
+from scipy.special import polygamma, gamma, digamma
+from scipy.special import beta as betafunction
+from fastbetabino import *
```

Comment:
I'd prefer not to rely on this `fastbetabinom` library since it appears to be be poorly maintained and has dependency on Cython. In particular I had trouble getting this library installed on my Mac.

Reply:
yes, must have kept it accidentally.